src-js/addOrbSection.js

```diff
@@ -0,0 +1,26 @@
+
+$(() => {
+  // https://app.optimizely.com/v2/projects/16812830475/experiments/20598037463/variations
+  window.OptimizelyClient.getVariationName({
+    experimentKey: 'dd_add_orb_section_to_docs_landing_page_test',
+    groupExperimentName: 'q3_fy22_docs_disco_experiment_group_test'
+  }).then(variation => {
+    if (variation === "treatment") {
+      $(".orb-bullet").hide();
+      $("#orb-section").show();
+
+      $("#orb-intro").click(function () {
+        window.AnalyticsClient.trackAction('docs-orb-bullet', { link: "Getting Started with Orbs" });
+      })
+      $("#orb-concepts").click(function () {
+        window.AnalyticsClient.trackAction('docs-orb-bullet', { link: "Core Concepts" });
+      })
+      $("#orb-publish").click(function () {
+        window.AnalyticsClient.trackAction('docs-orb-bullet', { link: "Publishing Your Orb" });
+      })
+      $("#orb-faq").click(function () {
+        window.AnalyticsClient.trackAction('docs-orb-bullet', { link: "Orbs FAQ" });
+      })
```

Comment:
This can be clean up by using a multiple selector:

```suggestion
      $("#orb-intro, #orb-concepts, #orb-publish, #orb-faq").click(function() {
        window.AnalyticsClient.trackAction('docs-orb-bullet', { link: this.id });
      });
```

Reply:
do we want to stick w jquery or do we want to try to use vanilla js where possible?

vanilla js is supposed to be more modern & performant.

Reply:
Originally it was vanilla js, but for some reason it wasnt loading the file. Since its an experiment we could switch it back to vanilla js if we decide to keep the section?

Reply:
I am open to discuss that but I would prefer if we keep jquery (or some other lighter library) that helps us with the overhead of dealing with vanilla JS. I'm generally okay with the performance hit using jQuery (or similar) because it brings stability in dealing with inconsistencies between all available browsers and especially because it provides a consistent API which I think is key to avoid bugs/weirdness.